docker/.env

```diff
@@ -46,6 +46,12 @@ REG_LOADER_IMAGE=nasapds/registry-loader
 # Registry Crawler Service and Registry Harvest CLI.
 HARVEST_DATA_DIR=./test-data/registry-harvest-data
 
+# don't change the directory unless you are debugging one of the component
+# if you debug one of the component, make this directory the same as the host's dir
+# MUST be an absolute path
+#CONTAINER_HARVEST_DATA_DIR=/data
+CONTAINER_HARVEST_DATA_DIR=/Users/loubrieu/git/registry/docker/test-data/registry-harvest-data/
```

Comment:
This dir path contains your user name.

Reply:
Thanks @ramesh-maddegoda I forgot to switch back the path to what it was before.